composer.json

```diff
@@ -11,7 +11,8 @@
         "voku/portable-utf8": "^5.4"
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "^2.18"
+        "friendsofphp/php-cs-fixer": "^2.18",
+        "phpunit/phpunit": "9.5.0"
```

Comment:
Should we float the version like we do for php-cs-fixer?

Reply:
Yes, good call. Done.